meta-mender-commercial/classes/mender-closed-source-tar-utils.bbclass

```diff
@@ -0,0 +1,40 @@
+
+def mender_closed_source_srcrev_from_src_uri(d, src_uri, repo_name):
+    pref_version = d.getVar("PREFERRED_VERSION")
+    if pref_version is not None and pref_version.find("-build") >= 0:
```

Comment:
Question:
Why not just `if prev_version and...`

Reply:
Removed also the `is not None` from the expressions below.